core/server/users.js

```diff
@@ -137,6 +137,7 @@ Meteor.methods({
       bio: Match.Optional(String),
       defaultReaction: Match.Optional(String),
       name: Match.Optional(String),
+      baseline: Match.Optional(String),
```

Comment:
Not really related to these changes but I think we're going to have to start watching the entries, at least string length

Reply:
Yes, I agree :)